provider/pkg/provider/access_token.go

```diff
@@ -20,10 +21,31 @@ type PulumiServiceAccessTokenInput struct {
 	Description string
 }
 
-func (i *PulumiServiceAccessTokenInput) ToPropertyMap() resource.PropertyMap {
-	pm := resource.PropertyMap{}
-	pm["description"] = resource.NewPropertyValue(i.Description)
-	return pm
+// AccessToken uses outdated way of storing input in internal __inputs property
+func GenerateProperties(input PulumiServiceAccessTokenInput, accessToken pulumiapi.AccessToken) (outputs *structpb.Struct, inputs *structpb.Struct, err error) {
+	inputMap := resource.PropertyMap{}
+	inputMap["description"] = resource.NewPropertyValue(input.Description)
+	outputStore := resource.PropertyMap{}
+	outputStore["__inputs"] = resource.NewObjectProperty(inputMap)
```

Comment:
Should we stop doing this `__inputs` thing like we did for webhooks: https://github.com/pulumi/pulumi-pulumiservice/pull/87

As described in the PR, [we don't do this __inputs thing in ay other providers](https://github.com/pulumi/pulumi-pulumiservice/pull/87#:~:text=We%20don%27t%20do%20this%20__inputs%20thing%20in%20any%20other%20providers) and have also been refactoring our code to move away from it in this repo.

Reply:
I can do that, was trying to keep refactor just to the Read method fix

Reply:
Totally cool to leave as-is for now, but let's open an issue to follow-up?

Reply:
Created https://github.com/pulumi/pulumi-pulumiservice/issues/342